### ISSUE TITLE
64 bit support is part of GLSL.std.450

### ISSUE THREAD

**Issue by danilw** [keith2018/spvm#26]
**I think this should be todo to add 64 bit support.**

> Specifications that the project follows is:
> GLSL.std.450

When:
> Only support 32-bits width Numerical type (float, integer)

https://registry.khronos.org/SPIR-V/specs/1.0/GLSL.std.450.html

You can see there listed functions that support only 16-32 bit:
![1](https://user-images.githubusercontent.com/24825887/190901140-f81c24e9-db77-4b65-b1b3-e581a7f36f1b.png)

And when it says "component type is floating-point" - function support all float types:
![2](https://user-images.githubusercontent.com/24825887/190901176-0e331c09-26f6-444e-9939-34c8e0f53759.png)

Without 64 bit support it downgrade to GLES 3.3, but you use SPIRV so...

**Comment by keith2018** [comment]
Yes, 64-bit numerical type should be supported, there are instructions works only for 64-bit float such as `PackDouble2x32`, `UnpackDouble2x32`
